### PR TITLE
Document sampling and publishing behavior

### LINKED ISSUE #218: Document sampling behavior.
New behavior was introduced in #217 that needs to be present in the readme/config file.

### PR DESCRIPTION
**Pull request by matthewloring**
Fixes #218

### CODE REVIEW

**Comment by matthewloring** [comment]
@ofrobots PTAL.

File: README.md [hunk 1]
```diff
@@ -106,6 +106,12 @@ The trace agent can be configured by passing a configurations object to the agen
 
 One configuration option of note is `enhancedDatabaseReporting`. Setting this option to `true` will cause database operations for redis and MongoDB to record query summaries and results as labels on reported trace spans.
 
+## Trace batching and sampling
+
+Batching behavior can be configured using the `flushDelaySeconds` and `bufferSize` options. The spans recorded for each incoming requests are placed in a buffer after the request has completed. Spans will be published to the UI in batch when the spans from `bufferSize` requests have been queued in the buffer or after `flushDelaySeconds` have passed since the last publish, whichever comes first.
```
Comment (ofrobots) [inline README.md]: Define batching. Add link to the configuration file because the configuration settings are documented in there.

Comment (matthewloring) [inline README.md]: Done.

**Comment by ofrobots** [comment]
lgtm.

**Pull request merged**